content/en/docs/tasks/inject-data-application/define-environment-variable-container.md

```diff
@@ -103,7 +103,7 @@ spec:
     - name: NAME
       value: "Kubernetes"
     command: ["echo"]
-    args: ["$(GREETING) $(HONORIFIC) $(NAME)"]
```

Comment:
I think the original is correct, and the next section immediately links to a page about using environment variables.